tests/tools/test_parmetis_wrapper.py

```diff
@@ -0,0 +1,86 @@
+import argparse
```

Comment:
as we talked, pls merge these 3 scripts into one and each script correspond a testcase.

Reply:
Done. The new file is called test_parmetis.py and will have 3 functions namely _test_parmetis_preprocessing(), _test_parmetis_wrapper() and test_parmetis_postprocessing(). The first two test cases will be skipped because they depend on mpirun. As per my discussion with Minjie on Thursday evening.